lib/id3c/cli/command/geocode.py

```diff
@@ -411,6 +412,34 @@ def parse_first_smartystreets_result(result: list) -> dict:
     return response
 
 
+def extract_address(address: dict) -> dict:
+    """
+    Given an *address*, converts it to text and returns a result from
+    the SmartyStreets US Extract API containing information about an address
+    connected to the text.
+
+    Note that this API is not consistent with the US Street API, and the lookup
+    and responses must be handled differently.
+    """
+    LOG.debug("Previous lookup failed. Looking up address as free text.")
+
+    client = smartystreets_client_builder().build_us_extract_api_client()
```

Comment:
For large batches of looksup, it'll be quick if we use a singleton copy of the client instead of building it for each request. This is what `geocode_address()` does, for example. The global `CLIENT` in that function will need to be renamed to be more specific.